components/member-profile/contribution/index.js

```diff
@@ -68,9 +74,11 @@ const Contribution = ({ contribution, fullName, imageLink, devUser }) => {
         </div>
       </div>
       <div className={classNames.featureLink}>
-        <a href={featureUrl || prList[0]['url']} target="_blank" rel="noreferrer">
-          Check out this feature in action
-        </a>
+        {url && (
+          <a href={url} onClick={(e) => e.stopPropagation()} target="_blank" rel="noreferrer">
```

Comment:
We need to rethink this:

If the featureURL/URL is in members, we want to use the `<Link>`, otherwise we want to use `<a>`

Reply:
You should read up on client-side routing vs routing through `a` tags. This is a good starting point: https://nextjs.org/docs/routing/introduction